### PR TITLE
[13.0][IMP] stock_request: add pivot report to stock request

### PR DESCRIPTION
**Pull request by albariera**
This image can be seen an example
Captura de pantalla de 2021-09-23 16-31-58
:

### CODE REVIEW

File: stock_request/views/stock_request_report.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                <field name="product_id" type="row" />
+                <field name="warehouse_id" type="col" />
+                <field name="location_id" type="col" />
+            </pivot>
+        </field>
+    </record>
+    <record id="stock_request_pivot_search" model="ir.ui.view">
+        <field name="name">stock.request.pivot.search</field>
+        <field name="model">stock.request</field>
+        <field name="arch" type="xml">
```
Comment (etobella) [inline stock_request/views/stock_request_report.xml:32]: Maybe you should inherit https://github.com/OCA/stock-logistics-warehouse/blob/13.0/stock_request/views/stock_request_views.xml#L38 and make this primary this way we can reuse the logic

Comment (albariera) [reply stock_request/views/stock_request_report.xml:32]: Done! Thanks :blush:

**Comment by kittiu** [comment]
Thanks for your work. Can you change the title to [13.0][IMP] xxx

**Comment by albariera** [comment]
@kittiu Sorry for the mistake. Thanks!

File: stock_request/views/stock_request_report.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <record id="stock_request_pivot_search" model="ir.ui.view">
+        <field name="name">stock.request.pivot.search</field>
+        <field name="model">stock.request</field>
+        <field name="inherit_id" ref="stock_request.stock_request_search" />
+        <field name="arch" type="xml">
+            <filter name="current_requests" position="after">
+                <separator />
+                <filter
+                    string="Date"
+                    name="month"
```
Comment (kittiu) [inline stock_request/views/stock_request_report.xml]: Date or month?
```suggestion
                    string="Date"
                    name="month"
```

**Review by kittiu — APPROVED** [review]

File: stock_request/views/stock_request_report.xml (hunk 2 above)
Comment (albariera) [reply stock_request/views/stock_request_report.xml]: I have changed all to date, but if you consider that "month" is a better option I can change it again :blush:

**Review by etobella — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM!

**Pull request merged**